### PR TITLE
zh-trans:Add manage-resources/memory-default-namespace.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans:Add manage-resources/memory-default-namespace.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    memory: 512Mi
+  requests:
+    memory: 128Mi
+```
+
+<!--
+## Motivation for default memory limits and requests
+-->
+
+## 设置内存限制和请求的动机
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: 已改，多谢！

File: content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+<!--
+If your namespace has a resource quota,
+it is helpful to have a default value in place for memory limit.
+Here are two of the restrictions that a resource quota imposes on a namespace:
+-->
+
+如果你的命名空间有资源配额，那么默认内存限制是很有帮助的。这里给出了为命名空间限定资源配额的两个限制规定：
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: 已改，多谢！

File: content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+{{% capture overview %}}
+
+<!--
+This page shows how to configure default memory requests and limits for a namespace.
+If a Container is created in a namespace that has a default memory limit, and the Container
+does not specify its own memory limit, then the Container is assigned the default memory limit.
+Kubernetes assigns a default memory request under certain conditions that are explained later in this topic.
+-->
+
+本章节介绍怎样给命名空间配置默认的内存请求和限制。如果在一个有默认内存限制的命名空间创建容器，该容器没有声明自己的内存限制时，将会被指定默认内存限制。Kubernetes 还为某些情况指定了默认的内存请求，本章后面会进行介绍。
```
Comment (markthink) [inline content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: 本章是个大的概念，原文没有这个意义..

Comment (AdamDang) [reply content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: 翻译成本页还是本文？

Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: 貌似差别不大，本页或本文都可以。

Comment (AdamDang) [reply content/zh/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: OK，那就本文

**Comment by tengqm** [comment]
@AdamDang CI gate job failed because 1) it failed to find yaml files referenced; 2) there could be something wrong in codenew shortcode.

Please help make copies of the yaml files you referenced from content/en/examples/admin/... to content/zh/examples/admin/... 

I'll be working on debugging the codenew shortcode at the same time.

Thanks.

**Comment by tengqm** [comment]
@AdamDang Just verified. If you put the referenced yaml files in the correct location, the CI gate will succeed. No upstream change or configuration settings are needed.

**Comment by AdamDang** [comment]
I've added all the yaml files in admin/resource directory.

**Comment by tengqm** [comment]
@AdamDang Don't do that. Please keep your each PR focused on one topic.

**Comment by AdamDang** [comment]
@tengqm 
So delete other yaml files?

**Comment by AdamDang** [comment]
OK now.

**Pull request merged**